include/fiction/technology/cell_ports.hpp

```diff
@@ -94,7 +94,8 @@ struct port_direction
         SOUTH,
         SOUTH_WEST,
         WEST,
-        NORTH_WEST
+        NORTH_WEST,
+        NONE
```

Comment:
This is an oversight on my end, but could you maybe add docstrings for the enum values here?